packages/loot-core/src/server/main.js

```diff
@@ -1839,10 +1834,6 @@ handlers['get-budgets'] = async function () {
   return budgets;
 };
 
-handlers['get-ynab4-files'] = async function () {
-  return YNAB4.findBudgets();
-};
```

Comment:
Did you mean to remove this? Seems to be still used in `getYNAB4Imports`.

Reply:
Didn't realise it was called by `getYNAB4Imports` but the problem is that `findBudgets` does not exist. So when I moved to ESM ESLint started complaining that such export was not there. What shall we do?

Reply:
Looks like it's not used either. So we can delete both of these things :)

And it looks like `SET_AVAILABLE_IMPORTS` won't be used either then. So some more potential for cleanups.